website/docs/language/settings/tf-cloud.mdx

```diff
@@ -5,7 +5,7 @@ description: >-
 
 # Cloud Configuration
 
-The main module of an OpenTofu configuration can integrate with a cloud backend to enable CLI-driven run workflow (if supported by your cloud backend). You only need to configure these settings when you want to use OpenTofu CLI to interact with a cloud backend.
+The main module of an OpenTofu configuration can integrate with a cloud backend to enable its [CLI-driven run workflow](docs/cli/cloud) (if supported by your cloud backend). You only need to configure these settings when you want to use OpenTofu CLI to interact with a cloud backend.
```

Comment:
Small nit

Align doc link to start with a `/`, as all other doc links are today in the docs

Reply:
Agreed, and added.